pgmpy/factors/__init__.py

```diff
@@ -2,6 +2,7 @@
 from .FactorSet import FactorSet, factorset_product, factorset_divide
 from .CPD import TabularCPD
 from .JointProbabilityDistribution import JointProbabilityDistribution
+from .continuous.ContinuousNode import ContinuousNode
```

Comment:
@khalibartan The tests are passing. Can you please check this too if you have doubts? I don't think there would be a problem.